.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+

```

Comment:
Should we remind people (or can we even provide the starter text for) the "Fixes #XXX" message? Those are really handy for whoever reviews and merges the PR because it (a) provides context for why the PR exists and (b) merging the PR also closes the related issue.

Reply:
Good idea will add. I take it for granted people do that but not everyone does.

Reply:
Added.